buildTool/webpack/testUnit/templates/webpack.testUnit.config.js.tpl

```diff
@@ -6,12 +6,12 @@ var outputFormat = buildJS.outputFormat;
 
 if (sourceFormat !== outputFormat) { -%>
 config.module.loaders.push({
-  test: /test\.files\.js$/,
+  test: /test\.files\.jsx?$/,
   loader: 'babel-loader',
   query: {
-    // https://github.com/babel/babel-loader#options
+// https://github.com/babel/babel-loader#options
     cacheDirectory: true,
-    presets: ['es2015']
+    presets: ['react','es2015']
```

Comment:
Not sure I'm clear on *how* to tie it to a framework choice... could I do something like:
```javascript
presets: [<% if (jsx) { %> 'react', <% } %> 'es2015']
```